### PR TITLE
:recycle: refactor: merge jwt middleware to contrib repo

### PR DESCRIPTION
**Pull request by efectn**
Same as https://github.com/gofiber/contrib/pull/596

This PR should be merged after v4 released on https://github.com/gofiber/jwt repo

### CODE REVIEW

**Comment by ReneWerner87** [comment]
thx, can you add a readme and go mod change in the jwt repository

**Comment by ReneWerner87** [comment]
- [ ] adjust the workflows (tests, dependabot, release drafter ... )

**Comment by efectn** [comment]
> * [x]  adjust the workflows (tests, dependabot, release drafter ... )

Done

**Comment by ReneWerner87** [comment]
- [ ] update dependabot yaml

File: jwt/go.mod [hunk 1]
```diff
@@ -0,0 +1,27 @@
+module github.com/gofiber/contrib/jwt
+
+go 1.20
```
Comment (gaby) [inline jwt/go.mod:3]: Is this suppose to be go1.20 ?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -58,3 +58,5 @@ jobs:
                 run: cd ./fiberi18n && go test ./... -v -race
             -   name: Test fiberzerolog Middleware
                 run: cd ./fiberzerolog && go test ./... -v -race
+            -   name: Test jwt Middleware
+                run: cd ./jwt && go test ./... -v -race
```
Comment (gaby) [inline .github/workflows/test.yml:62]: While we are at it, can you add go1.20 to the test matrix

**Review by gaby — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 2 above)
Comment (ReneWerner87) [reply .github/workflows/test.yml:62]: solved in websocket PR https://github.com/gofiber/contrib/pull/596

**Review by ReneWerner87 — APPROVED** [review]

**Pull request merged**